server/api/org/org.model.ts

```diff
@@ -44,4 +50,26 @@ export class Org extends BaseEntity {
     default: '[]',
   })
   defaultMusterConfiguration: MusterConfiguration[] = [];
+
+  async getUsers() {
+    const orgUsers: User[] = [];
+
+    const roles = await Role.find({
```

Comment:
This logic is actually flawed as it exposes user roles for other orgs that the caller may not have access to.  Take a look at #104 for how this is fixed.  It would actually be a good test to check that each user in the response has exactly one role for the current org.

Reply:
Cool. I rebased and used the latest code for this function.